src/core/thread/mle.cpp

```diff
@@ -2101,10 +2108,11 @@ ThreadError Mle::HandleChildIdResponse(const Message &aMessage, const Ip6::Messa
     SuccessOrExit(error = Tlv::GetTlv(aMessage, Tlv::kNetworkData, sizeof(networkData), networkData));
 
     // Active Timestamp
-    if (Tlv::GetTlv(aMessage, Tlv::kActiveTimestamp, sizeof(activeTimestamp), activeTimestamp) == kThreadError_None)
-    {
-        VerifyOrExit(activeTimestamp.IsValid(), error = kThreadError_Parse);
+    SuccessOrExit(error = Tlv::GetTlv(aMessage, Tlv::kActiveTimestamp, sizeof(activeTimestamp), activeTimestamp));
```

Comment:
The spec seems to indicate that the Active Timestamp TLV is not required in the Child ID Response message.  In particular, if the Child ID Request indicates that the joining child already has the proper Active Timestamp, then the Child ID Response does not need to include the Active Timestamp TLV.